apps/portals/b2ai.standards/src/pages/HomePage.tsx

```diff
@@ -18,7 +24,8 @@ export default function HomePage() {
           Standards Related to Bridge2AI Challenges
         </h2>
       </div>
-      <CardDeck
+      <ChallengesCardDeck />
```

Comment:
Looks like the ChallengesCardDeck and TableQueryCardDeck produce duplicate output (ChallengesCardDeck is currently a subset, but can be extended).  Is that correct?  And if so, should this determination be behind a feature flag, or ChallengesCardDeck be commented out?

Reply:
This was an example, I'll remove the new one to not 'break' staging, but @Sigfried can add it back when they get it working how they want.